api/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -1011,6 +1011,8 @@ type GlobalConfig struct {
 	// +listType=set
 	Tags []string `json:"tags,omitempty"`
 
+	Env []corev1.EnvVar `json:"env,omitempty"`
```

Comment:
```suggestion
	// Set environment variables for all Agents.
	// +optional
	// +listType=map
	// +listMapKey=name
	Env []corev1.EnvVar `json:"env,omitempty"`
```
Could we please add a comment here explaining what this new option does ? I took the wording from the Helm chart as an example.
This is to ensure `docs/configuration.v2alpha1.md` gets populated.

Reply:
Adding a comment here also modified `config/crd/bases/v1/datadoghq.com_datadogagents.yaml` when I ran `make generate`, should I also include that change?